### PR TITLE
[util] Add entropy tests to strong_random.py

### PR DESCRIPTION
**Pull request by vrozic**
Adds tests to evaluate statistical quality of the entropy buffer. These tests are triggered when the entropy buffer is loaded from a file and report an error if any of the tests fails.

### CODE REVIEW

File: util/topgen/strong_random.py [hunk 1]
```diff
@@ -45,6 +194,18 @@ def load(self, input_file):
...
+        # the buffer size must be a multiple of SEQ_SIZE_BYTES bytes.
+        if self.size() % self.SEQ_SIZE_BYTES:
+            log.error("The file size must be a multiple of 20000 bits.")
+            sys.exit(1)
+        # Testing statistical quality of the loaded buffer. The quality is
+        # sufficient only if all tests pass.
+        all_tests_pass = self.test_monobit() and self.test_poker() and \
+            self.test_runs() and self.test_long_run()
+        if not all_tests_pass:
+            log.error("Insuficient entropy in the buffer. Some test fail.")
```
Comment (abdullahvarici) [inline util/topgen/strong_random.py:208]: Wouldn't it be better to report which tests fail?

**Review by abdullahvarici — APPROVED** [review]
Thanks @vrozic. I also checked the test descriptions in the AIS-31 document and I couldn't find any mismatch between the code and the test descriptions. LGTM!

File: util/topgen/strong_random.py (hunk 1 above)
Comment (msfschaffner) [reply util/topgen/strong_random.py:208]: +1

**Review by msfschaffner — APPROVED** [review]
A few minor comments, otherwise LGTM

File: util/topgen/strong_random.py (hunk 1 above)
Comment (vrozic) [reply util/topgen/strong_random.py]: Agreed
I've modified the error message to report exactly which tests fail.

**Comment by vrozic** [comment]
Thanks @abdullahvarici  and @msfschaffner for the comments.
I've made the suggested changes.

**Pull request merged**